### DISCUSSION TITLE
Run on Aws Lightsail

### DISCUSSION THREAD

**Discussion by Idan4Winds** [gotenberg/gotenberg#458] [Q&A]
My deployment just dont work .. can someone help here? 
 **my config :** 
![Screen Shot 2022-05-10 at 14 22 18](https://user-images.githubusercontent.com/79466240/167617625-cb3a1ea6-2782-4727-a2ae-f20a94981b98.png)
**Error log :** 
![Screen Shot 2022-05-10 at 14 25 31](https://user-images.githubusercontent.com/79466240/167618016-1c0ddf65-6523-424e-a98a-b1a1b970e394.png)

**Comment by Idan4Winds** [comment]
Update Working with 7.4.3 👍

**Reply by gulien** [reply]
Looks like your health check is on the `/` route, which does not exist. I wonder why it is working with `7.4.3` though 🤔
